warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -533,6 +559,10 @@ public void execMulti(String script) throws WarpScriptException {
 
   @Override
   public void exec(String line) throws WarpScriptException {
```

Comment:
This may negatively impact performance by adding a function call before inlining so you should assess actual performance impact  through a micro benchmark and decide if calls to exec(line) should be replaced by exec(line, -1) everywhere.

Reply:
WarpRun and lots of other functions uses execMulti, so I updated it.
RunTests, loadBootstrap, WarpScriptExecutor, INCLUDE also loop on exec(), so I updated them.

Question remains for EgressInteractiveHandler.

Reply:
EgressInteractiveHandler is out of scope when it comes to line numbers